metricbeat/templates/secrets.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: {{ .Values.apiVersions.Secrets }}
+apiVersion: v1 {{ .Values.apiVersions.Secrets }}
```

Comment:
@yyyogev 
Is this change on purpose?
It's in file in the metricbeat folder, and the placeholder seems out of place

Reply:
no, a mistake (I think goland did this for some reason)